mlflow/langchain/utils.py

```diff
@@ -263,6 +264,13 @@ def _validate_and_wrap_lc_model(lc_model, loader_fn):
     import langchain.llms.openai
     import langchain.schema
 
+    if isinstance(lc_model, str):
+        if os.path.basename(os.path.abspath(lc_model)) != "chain.py":
+            raise mlflow.MlflowException.invalid_parameter_value(
+                f"The path to the chain.py file must be provided, but provided {lc_model}."
```

Comment:
Can we be a bit more explicit and say something like "If `lc_model` is a string, it must be a path to a file called `chain.py` on the local filesystem?"